central/complianceoperator/v2/report/manager/complianceReportgenerator/consts.go

```diff
@@ -0,0 +1,13 @@
+package complianceReportgenerator
+
+const (
+	defaultEmailBodyTemplate = "{{.BrandedPrefix}} has identified {{.ComplianceStatus}} profile checks for clusters scanned by your \n" +
+		"schedule configuration parameters. The attached report lists those checks and associated details to aid with remediation. \n" +
+		"Profiles:{{.Profile}} \n" +
+		"Passing:{{.Pass}} checks \n" +
+		"Failing:{{.Fail}} checks \n" +
+		"Mixed:{{.Mixed}} checks \n" +
+		"Clusters {{.Clusters}} scanned"
+
+	defaultSubjectTemplate = "{{.BrandedPrefix}} Compliance Report For {{.ScanConfig}} with {{.Profiles}} Profiles"
```

Comment:
Total number of profiles instead of profile names? The two outstanding PRs are a bit confusing, so appreciate your patience with repeated comments!

Reply:
yes subject will be "{{.BrandedPrefix}} Compliance Report For <scan config name> with 4 Profiles"